app/code/Transoft/Blog/etc/acl.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:framework:Acl/etc/acl.xsd">
+    <acl>
+        <resources>
+            <resource id="Magento_Backend::admin">
+                <resource id="Transoft_Blog::post" title="Blog Title"
```

Comment:
Please rename **Blog Title** to **Blog Posts**

Reply:
renamed